idl/sprot.idol

```diff
@@ -129,5 +129,26 @@ Interface(
                 err: CLike("SprotError"),
             ),
         ),
+        "reset_component": (
```

Comment:
(I recommend using `encoding: Hubpack` for all new IPC operations. This would let you remove a lot of the lines below.)